portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,35 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a favorite quote to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addFavoriteQuote() {
+  const quotes =
+      ["Many years later, as he faced the firing squad, Colonel Aureliano Buendía was to remember that distant afternoon when his father took him to discover ice. \n —Gabriel García Márquez, One Hundred Years of Solitude",
+      "Hope is a dangerous thing. Hope can drive a man insane. \n - The Shawshank Redemption",
+      "Bran thought about it. 'Can a man still be brave if he's afraid?' 'That is the only time a man can be brave,' his father told him. \n ― George R.R. Martin, A Game of Thrones"];
```

Comment:
It's good to keep things consistent across the code and comments so I'd suggest renaming all of these to "favoriteQuotes" and "favoriteQuote" everywhere, or just have them as "quotes" and "quote" in all places. But the idea is to stick to one.

Reply:
Got it.